packages/openneuro-app/src/scripts/errors/errorBoundary.jsx

```diff
@@ -1,8 +1,50 @@
 import * as Sentry from '@sentry/browser'
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Modal } from '../utils/modal.jsx'
 import FreshdeskWidget from '../datalad/fragments/freshdesk-widget.jsx'
+import styled from '@emotion/styled'
+
+//styled components
+
+const Overlay = styled.div`
+  background: rgba(0, 0, 0, 0.8);
+  padding-top: 55px;
+  pointer-events: none;
+  bottom: 0;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  -webkit-transition: opacity 400ms ease-in;
+  -moz-transition: opacity 400ms ease-in;
+  transition: opacity 400ms ease-in;
+  display: inline-block;
+  opacity: 100;
+  z-index: 99;
+`
+
+const ModalContainer = styled.div`
+  background-color: white;
+  height: 100%;
+  width: 100%;
+  position: relative;
+  margin: 0 auto;
+  padding: 3em;
+  pointer-events: auto;
+`
+
+const ExitButton = styled.a`
+  color: black;
+  font-size: 34px;
+  padding: 12px 12px;
+  padding-top: 20px;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  pointer-events: auto;
+  top: 0;
+  z-index: 100;
+`
```

Comment:
These style components should probably be their own module, particularly since they are imported in both variants of the support modal and will likely see some reuse elsewhere in the future.

Reply:
Good idea; I'll update in just a sec

Reply:
Agreed, and it'd be convenient for them to be packaged together as a single Modal wrapper component.

Reply:
@david-nishi Could do that, but don't see where a wrapper would warrant the opportunity cost ( children, various props would still need to be called/passed) Also, theming requires a separate library. Might be misunderstanding you though (do lmk if I am)